app/models/page.rb

```diff
@@ -16,6 +16,10 @@ def self.top_level_hidden_pages
     Page.where( section: nil, hidden: true )
   end
 
+  def self.are_there_any_hidden_pages?
+    Page.where( hidden: true ).count.positive?
```

Comment:
`exists?` also works for these kind of queries:

```diff
-Page.where( hidden: true ).count.positive?
+Page.exists?(hidden: true)
```

https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-exists-3F